packages/web-react/src/components/Text/stories/Text.stories.tsx

```diff
@@ -17,12 +17,19 @@ const meta: Meta<typeof Text> = {
     children: {
       control: 'text',
     },
+    color: {
+      control: 'select',
+      options: [...Object.values(TextColors), undefined],
+      table: {
+        defaultValue: { summary: undefined },
+      },
+    },
     elementType: {
       control: 'text',
     },
     emphasis: {
       control: 'select',
-      options: ['italic', 'bold', 'semibold', undefined],
+      options: [...Object.values(Emphasis), Emphasis],
```

Comment:
```suggestion
      options: [...Object.values(Emphasis)],
```
I think you have there extra Emphasis

Reply:
There should still be the undefined. :) Thanks!